src/runtime/entries/cloudflare-pages.ts

```diff
@@ -21,9 +21,11 @@ interface CFRequestContext {
 export async function onRequest(ctx: CFRequestContext) {
   try {
     // const asset = await env.ASSETS.fetch(request, { cacheControl: assetsCacheControl })
-    const asset = await ctx.next();
-    if (asset.status !== 404) {
-      return asset;
+    if (ctx.request.method === "GET") {
```

Comment:
I think we might also support for HEAD 
```suggestion
    if (ctx.request.method === "GET" || ctx.request.method === "HEAD") {
```

Reply:
No, that is not allowed by Cloudflare either.

Reply:
Hmm thanks for confirming.

This is strange because normal worker has fine gained API to control caching headers via head. (and i suppose would be useful for cors and options)

Reply:
It would indeed be useful. Maybe they can add more options down the line to configuration via `_routes.json`?

Reply:
Okay, let's revise it later GET is good enough.